src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -217,13 +217,21 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
         .value;
       latestTokenPrice.value = (await data.next()).value["price"];
 
+      let previousTime = "1970-01-01T00:00:00Z";
+      let recentTime;
       for await (const value of data) {
         if (value.action === "CREATE") {
           if (value.type === "BALANCE") {
-            currentBalanceOfToken.value = convertWeiToQuantity(
-              value.result["walletValue"],
-              parseInt(decimals),
-            );
+            recentTime = value.result["timestamp"];
```

Comment:
IMO we could move let recentTime (if not used beyond the `for await` loop), to the 225 line  and we would get 
`const recentTime = new Date(timestamp).getTime();`

Reply:
done + i changed name of variables for better readability